lms/djangoapps/courseware/model_data.py

```diff
@@ -24,6 +26,20 @@
 from xmodule.modulestore.django import modulestore
 from xblock.core import XBlockAside
 
+try:
+    DJANGO_FIELD_CACHE = get_cache('xblock-fields')
+except InvalidCacheBackendError:
+    DJANGO_FIELD_CACHE = get_cache('default')
+
+DJANGO_CACHED_FIELDS = [
+    ('sequential', 'position'),
+    ('chapter', 'position'),
+    ('problemset', 'position'),
+    ('videosequence', 'position'),
+    ('video', 'speed'),
+    ('video', 'saved_video_position'),
+]
+
```

Comment:
So this change will set/get these values from the Django cache. Could they all be wiped from the cache without causing critical problems with the courseware? Should this change be accompanied with a wipe of the existing values in the DB for these fields to avoid future unexpected behavior?